test/k8sT/Services.go

```diff
@@ -432,6 +432,84 @@ var _ = Describe("K8sServicesTest", func() {
 		})
 	})
 
+	SkipContextIf(helpers.RunsWithKubeProxy, "Checks connectivity when skipping socket lb in pod ns", func() {
```

Comment:
My next question is, how do we handle this test on 4.19 kernels in CI if that particular kernel doesn't have the necessary helper to implement the functionality proposed here?

Reply:
Yea, I modified the condition function to skip on 4.19 kernels too  :)